tests/Common/BranchComponentTest.php

```diff
@@ -237,7 +237,7 @@ function ($branch) use ($branchName) {
         $this->assertEquals('dev-1-row-3', $rows[2]['id']);
         $this->assertCount(3, $rows);
 
-        // all version should be 1 until we implement versioning for dev branch
+        // updated rows should has version 1, if rows wasn't updated version should be 0
```

Comment:
rows = množné číslo => have (has jen 3.os. jednotného čísla), to samé "wasn't" => "weren't/were not" JFYI